packages/instantsearch.js/src/templates/carousel/carousel.tsx

```diff
@@ -66,12 +68,14 @@ export function carousel<TObject extends Record<string, unknown>>({
     items,
     templates: widgetTemplates,
     cssClasses: widgetCssClasses = {},
+    sendEvent = () => {},
```

Comment:
A hack to make TypeScript happy

Reply:
With `sendEvent` typed from the UI component you should be able to remove the default value, I think?

```suggestion
    sendEvent,
```